rest-api/dao/physical_measurements_dao.py

```diff
@@ -284,12 +292,21 @@ def _update_participant_summary(self, session, obj):
     participant_summary.physicalMeasurementsFinalizedTime = obj.finalized
     participant_summary.physicalMeasurementsCreatedSiteId = obj.createdSiteId
     participant_summary.physicalMeasurementsFinalizedSiteId = obj.finalizedSiteId
+    if 'physicalMeasurementsStatus' in obj:
```

Comment:
This logic looks wrong... you'll set participant_summary.physicalMeasurementStatus to COMPLETED here, and then the if block below that updates enrollment status won't get run?

We should make sure that update_enrollment_status gets run whenever the physical measurement status changes... that's what the previous logic was trying to accomplish.